commands.py

```diff
@@ -0,0 +1,46 @@
+from LSP.plugin.core.typing import List
+from os import path
+import sublime
+import sublime_plugin
+
+
+CONFIGURATION_FILENAME = 'pyrightconfig.json'
+CONFIGURATION_CONTENTS = '''{
+\t// Install LSP-json to get validation and autocompletion in this file.
```

Comment:
Maybe without this line containing LSP-json?

Reply:
It's good to advertise it since it can result in less support questions.
The user can alway remove it.